test/integration/meta_tags_test.rb

```diff
@@ -2,35 +2,35 @@
 
 class MetaTagsTest < ActionDispatch::IntegrationTest
   test "correct meta tags are displayed for pages" do
-    case_study = GovukSchemas::RandomExample.for_schema(frontend_schema: "news_article") do |random|
+    news_article = GovukSchemas::RandomExample.for_schema(frontend_schema: "news_article") do |random|
```

Comment:
This should be `detailed_guide` but you've already added this in the removal of news article rendering in https://github.com/alphagov/government-frontend/pull/3618/commits/cbf103ea17f1022e738afb1cd024621a59bb5431 but I'm guessing this PR would be merged first?

Reply:
Now that news_article is done, we should probably update here just for these two tests. I'll update it.